src/Core/Mews.Fiscalizations.Core/Mews.Fiscalizations.Core.csproj

```diff
@@ -11,13 +11,13 @@
     <RepositoryUrl>https://github.com/MewsSystems/fiscalizations</RepositoryUrl>
     <Icon>https://raw.githubusercontent.com/msigut/eet/master/receipt.png</Icon>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <PackageVersion>5.0.3</PackageVersion>
+    <PackageVersion>6.0.0</PackageVersion>
   </PropertyGroup>
 
   <ItemGroup>
     <None Include="..\..\..\LICENSE">
       <Pack>True</Pack>
-      <PackagePath></PackagePath>
+      <PackagePath>\</PackagePath>
```

Comment:
This fixes the license warning from nuget? 😄

Reply:
I hope so 😄 will see when the publish action runs